crates/ruma-events/src/room/avatar.rs

```diff
@@ -56,6 +56,7 @@ impl AvatarEventContent {
 
 /// Metadata about an image (specific to avatars).
 #[derive(Clone, Debug, Deserialize, Serialize)]
+#[cfg_attr(not(feature = "unstable-exhaustive-types"), non_exhaustive)]
 pub struct ImageInfo {
```

Comment:
Can you add a `Default` impl and a constructor fn that delegates to it? It seems like this would otherwise no longer be constructible (except through `Deserialize`).